packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -74,6 +74,9 @@ export class EditorToolbarComponent {
 	 * Opens the block inserter.
 	 */
 	async openBlockInserter(): Promise< void > {
+		// const toolbarLocator = this.frameLocator.locator( panel );
+		// await toolbarLocator.click( { position: { x: 100, y: 0 } } );
+
```

Comment:
FYI these commented lines are still in here!

Reply:
Argh